opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -258,9 +261,9 @@ def _check_attribute_value_sequence(sequence: Sequence) -> Optional[str]:
         if first_element_type not in (bool, str, Number):
             return "invalid type"
 
-        for element in sequence:
-            if not isinstance(element, first_element_type):
-                return "different type"
+        if any(not isinstance(element, first_element_type) for element in sequence):
+            return "different type"
```

Comment:
This looks like a matter of taste, probably shouldn't be included in this PR.

Reply:
have removed this part. thanks:)